cmd/orb-server/main.go

```diff
@@ -7,173 +7,25 @@ SPDX-License-Identifier: Apache-2.0
 package main
 
 import (
-	"context"
-	"fmt"
-	"os"
-	"os/signal"
-	"strings"
-	"syscall"
+	"github.com/spf13/cobra"
+	"github.com/trustbloc/edge-core/pkg/log"
 
-	"github.com/sirupsen/logrus"
-	"github.com/spf13/viper"
-	"github.com/trustbloc/sidetree-core-go/pkg/batch"
-	"github.com/trustbloc/sidetree-core-go/pkg/dochandler"
-	"github.com/trustbloc/sidetree-core-go/pkg/processor"
-	"github.com/trustbloc/sidetree-core-go/pkg/restapi/diddochandler"
-
-	sidetreecontext "github.com/trustbloc/orb/pkg/context"
-	"github.com/trustbloc/orb/pkg/context/cas"
-	"github.com/trustbloc/orb/pkg/context/txnclient"
-	"github.com/trustbloc/orb/pkg/didtxnref/memdidtxnref"
-	"github.com/trustbloc/orb/pkg/httpserver"
-	"github.com/trustbloc/orb/pkg/mocks"
-	"github.com/trustbloc/orb/pkg/observer"
-	"github.com/trustbloc/orb/pkg/txngraph"
+	"github.com/trustbloc/orb/cmd/orb-server/startcmd"
 )
 
-var logger = logrus.New()
-
-var config = viper.New()
-
-const (
-	defaultDIDDocNamespace = "did:orb"
-	basePath               = "/sidetree/0.0.1"
-	arrayDelimiter         = ","
-	txnBuffer              = 100
-)
+var logger = log.New("orb-rest")
```

Comment:
We should rename rest to server throughout.

Reply:
sure, next PR